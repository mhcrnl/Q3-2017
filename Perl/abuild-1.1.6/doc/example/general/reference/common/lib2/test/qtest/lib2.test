#!/usr/bin/env perl

require 5.008;
BEGIN { $^W = 1; }
use strict;

chdir("lib2") or die "chdir testdir failed: $!\n";

require TestDriver;

my $td = new TestDriver('lib2');

$td->runtest("test lib2 class",
	     {$td->COMMAND => "lib2_test"},
	     {$td->FILE => "lib2.out", $td->EXIT_STATUS => 0},
	     $td->NORMALIZE_NEWLINES);

$td->report(1);
