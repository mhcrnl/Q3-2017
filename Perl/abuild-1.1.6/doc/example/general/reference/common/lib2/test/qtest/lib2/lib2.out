not counting to -3
What do you find interesting about -3?
counting to 5: 1 2 3 4 5
What do you find interesting about 5?
